terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_db_subnet_group" "rds" {       
+  name = "default-db"                        
+  description = "default subnets"            
+  subnet_ids = ["${aws_subnet_wpt_id}"]
+}
+
+resource "aws_subnet_wpt" "wpt-subnet" {
+  vpc_id            = "vpc-eb65f992"
+  cidr_block        = "10.100.0.0/16"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_db_instance" "rds" {
+  identifier = "web-platform-tests-db"
+  storage_type = "gp2"
+  allocated_storage = 10
+  engine = "postgres"
+  engine_version = "9.5.4"
+  instance_class = "db.t2.small"
+  name = "wptdash"
+  username = "wptdash"
+  password = "${trimspace(file("${path.module}/secrets/wpt-db-password.txt"))}"
+  multi_az = true
+  backup_retention_period = 30
+  backup_window = "05:00-18:30"
+  maintenance_window = "sun:03:00-sun:04:00"
+  allow_major_version_upgrade = true
+  db_subnet_group_name = "${aws_db_subnet_group.rds.id}"
+  vpc_security_group_ids = [
+    "${aws_security_group.web-platform-tests-db1.id}",
+  ]
+}
+
+
+resource "aws_security_group" "web-platform-tests-db1" {  
+  name = "web-platform-tests-db1"
+
+  description = "RDS postgres servers (terraform-managed)"
+  vpc_id = "vpc-eb65f992"
```

Comment:
Don't hardcode the vpc, you can grab that with
`${module.vpc.id}`

That is available because the vpc module outputs it here:
https://github.com/bocoup/infrastructure-web-platform/blob/master/terraform/modules/aws/vpc/main.tf#L7

...and is consumed by this project here:
https://github.com/bocoup/infrastructure-web-platform/blob/master/terraform/projects/pr-dashboard/network.tf#L28-L32